advanced/Scripts/piholeDebug.sh

```diff
@@ -907,7 +907,7 @@ parse_file() {
         #shellcheck disable=SC2016
         IFS=$'\r\n' command eval 'file_info=( $(cat "${filename}") )'
     else
-        read -a file_info <<< $filename
+        read -a -r file_info <<< "$filename"
```

Comment:
This needs to be `read -r -a file_info <<< "$filename"`

`-a` takes an array as an argument.